### PR TITLE
Finishing quiz edition (QuestionActivity part) Sprint9/issue#113

### PR DESCRIPTION
**Pull request by nathan96g**

### CODE REVIEW

File: app/src/main/java/ch/epfl/qedit/view/edit/EditQuestionActivity.java [hunk 1]
```diff
@@ -47,48 +56,108 @@ protected void onCreate(Bundle savedInstanceState) {
...
+                    editFieldFragment.show(getSupportFragmentManager(), SOL_DIALOG_TAG);
                 });
     }
 
-    /** Remove when the real question is ready */
-    private void setupDummyResult() {
-        title = stringPool.add("This is a new title");
-        text = stringPool.add("This is a new text");
-        answerFormat = MatrixFormat.singleField(MatrixFormat.Field.textField("", 25));
+    /** This function is call in the dialog open when we click on a solution button */
```
Comment (Anthony-Iozzia) [inline app/src/main/java/ch/epfl/qedit/view/edit/EditQuestionActivity.java:111]: "called in the open dialog"

File: app/src/main/res/layout/activity_edit_question.xml [hunk 2]
```diff
@@ -45,40 +50,68 @@
...
-                        app:srcCompat="@drawable/ic_add_black" />
-                </TableRow>
-
-            </TableLayout>
-            <Button
-                android:id="@+id/button_done_editing"
+                android:layout_gravity="center"
+                android:orientation="horizontal">
+
+                <ImageButton
```
Comment (brunnerant) [inline app/src/main/res/layout/activity_edit_question.xml:62]: We should maybe add some visual feedback when the user has defined an answer format. Right now there is no information that tells you that you have successfully entered an answer format.

File: app/src/main/res/layout/fragment_edit_field.xml [hunk 3]
```diff
@@ -13,47 +13,56 @@
...
+
+            <LinearLayout
+                android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:entries="@array/field_types_list"/>
+                android:layout_marginTop="@dimen/regular_padding"
+                android:orientation="horizontal">
+
+                <TextView
+                    android:id="@+id/selectTypeText"
```
Comment (brunnerant) [inline app/src/main/res/layout/fragment_edit_field.xml:39]: I think that we have made the convention to use `snake_case` for the ids in xml files, so we should try to stick to it. Same for `signCheckBox`.

**Comment by brunnerant** [comment]
I have tested the edition experience on my phone, and it looks good to me, apart from the comments that I have made.

File: .idea/misc.xml [hunk 4]
```diff
@@ -1,5 +1,44 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="NullableNotNullManager">
```
[the remark this answers is not in the document]
Comment (nathan96g) [reply .idea/misc.xml:3]: To be perfectly honnest, i don't know! everythings was added automatically

File: app/src/main/res/layout/activity_edit_question.xml (hunk 2 above)
Comment (nathan96g) [reply app/src/main/res/layout/activity_edit_question.xml:62]: Thanks, i just did that!

File: app/src/main/res/layout/fragment_edit_field.xml (hunk 3 above)
Comment (nathan96g) [reply app/src/main/res/layout/fragment_edit_field.xml:39]: oh yes, thanks!

**Pull request merged**